hyperspy/signal.py

```diff
@@ -4051,6 +4066,10 @@ def min(self, axis, out=None):
         axis : {int | string}
            The axis can be specified using the index of the axis in
            `axes_manager` or the axis name.
+        out : {Signal, None}
```

Comment:
Wouldn't it be better to add the `out` documentation to `hyperspy/doctrings`. See [e.g. here](https://github.com/hyperspy/hyperspy/blob/master/hyperspy/__init__.py#L31).